sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOTest.java

```diff
@@ -404,6 +406,37 @@ public void testCompressedRead() throws Exception {
 
   @Test
   @Category(NeedsRunner.class)
```

Comment:
@jbonofre nope, no changes needed in this PR; not asking anything from you.

For now, just trying to seed a conversation about the `@Category` tag in general. Specifically, "needs runner" is not currently descriptive enough. This test wouldn't run in Cloud Dataflow and probably also not in a Flink or Spark cluster -- it relies on the local file system.